src/sb_utils/test/rviz_utils_rostest.cpp

```diff
@@ -0,0 +1,211 @@
+#include "RvizUtils.h"
+#include <gtest/gtest.h>
+
```

Comment:
Quick comment on why this is a RosTest (I presume because of timestamps and whatnot) would be good here

Reply:
bump?

Reply:
Should already be done, it should be at the top of the file, not sure why it's not showing up here.